README.md

```diff
@@ -42,9 +52,15 @@ If you have any questions about this process, please use the internet as a resou
 
   - Commit often (with useful commit messages). This is best practice and it means you can pick-up where you left-off or revert to a previous state easily.
   - Push somewhat often. This lets you 'save' your work remotely, so anyone else can see what you are doing and you won't lose everything if your computer burns or drowns.
+  - Merge any updates to the feature you are working on if changes have in the develop branch using the following steps:
```

Comment:
"Merge any updates to the feature you are working on if changes have in the develop branch using the following steps:"
 I think this needs to be rephrased a little.
Maybe start with 'if the develop branch has changed since you first started working on your feature, "